### ISSUE TITLE
React component name not matching the file name in the Newsfeed tutorial

### ISSUE THREAD

**Issue by joeccleung** [relayjs/relay-examples#286]
Hello, I am following the Newsfeed tutorial. In the tutorial, the React component "OrganizationKind" default method was named _PosterBylineLocation_ instead of _OrganizationKind_.

https://github.com/relayjs/relay-examples/blob/d2c840366dab237d9e6dab4caf00f88d7dd870c3/newsfeed/src/components/OrganizationKind.tsx#L19

I am not sure is this by design or a typo.

BTW, thank you for the great tutorial.

**Comment by davidmccabe** [comment]
Yeah this is just a mistake. Thanks for offering to fix it, please go ahead!
